nautobot/dcim/elevations.py

```diff
@@ -217,7 +221,93 @@ def merge_elevations(self, face):
 
         return elevation
 
-    def render(self, face, unit_width, unit_height, legend_width):
+    def render(
+        self,
+        fileformat: Literal["svg", "csv"],
```

Comment:
Similarly, can we preserve the existing API signature by making `fileformat` an optional kwarg (defaulting to `"svg"`) instead of a new mandatory arg?